djando_documentation.md

```diff
@@ -0,0 +1,17 @@
+#### Django
+
+Django is a free and open source web application framework that is written in Python and helps to implement websites in a convenient manner. With the collection of modules embedded within its framework, development is made easier and one would not have to work on creating the website from scratch but just use the components that are already built within which saves a lot of time. Acknowledging the fact that we have a short span of time to work on the GatorGrouper project, we decided to use Django as it provides the developer with simpler tools to grasp the project's structure. In addition to this, it supports Object-Relational Mapping, which allows the user to create a bridge between the data model and the database engine, and supports a large set of database systems including SQLite.
+
+#### Running
+
+After the successful installation of Django, we inclined towards running the application. In order to run the application smoothly, please make sure that you are in the correct directory which is GatorGrouper_Django.
```

Comment:
"inclined towards" doesn't make any sense.

Additionally, `GatorGrouper_Django` is no longer the correct folder.